slasher/db/kv/spanner_new.go

```diff
@@ -0,0 +1,110 @@
+package kv
+
+import (
+	"context"
+	"errors"
+
+	"github.com/prysmaticlabs/prysm/shared/bytesutil"
+	"github.com/prysmaticlabs/prysm/slasher/detection/attestations/types"
+	bolt "go.etcd.io/bbolt"
+	"go.opencensus.io/trace"
+)
+
+var errWrongSize = errors.New("wrong data length for min max span byte array")
+var highestObservedValidatorIdx uint64
+
+// GetValidatorSpan unmarshal a span from an encoded, flattened array.
+func (db *Store) GetValidatorSpan(ctx context.Context, spans []byte, validatorIdx uint64) (types.Span, error) {
```

Comment:
Same for this function, can it be moved out of `db`? Accessing this from `db.GetValidatorSpan` could be misleading

Reply:
moved it as suggested